### PR TITLE
[added] pagination boundaryLinks

### PR DESCRIPTION
**Pull request by Spotinux**

### CODE REVIEW

**Comment by taion** [comment]
Mind adding some test coverage here?

**Comment by AlexKVal** [comment]
Before:
before

After:
after

@Spotinux please squash commits, otherwise LGTM :sparkle: 
and it needs one more @react-bootstrap/collaborators review.

File: docs/examples/PaginationAdvanced.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const PaginationAdvanced = React.createClass({
         ellipsis
         items={20}
         maxButtons={5}
+        boundaryLinks
```
Comment (mathieumg) [inline docs/examples/PaginationAdvanced.js]: Perhaps place it beside the other booleans?

File: src/Pagination.js [hunk 2]
```diff
@@ -128,6 +158,21 @@ const Pagination = React.createClass({
           </span>
         </PaginationButton>
       );
+
+      if (boundaryLinks) {
+        if (endPage !== items) {
```
Comment (mathieumg) [inline src/Pagination.js]: Any reason why the two `if`s couldn't be combined?

**Comment by mathieumg** [comment]
LGTM

**Comment by taion** [comment]
@Spotinux Can you rebase + squash this PR real quick? The diff has some pollution on it.

**Comment by Spotinux** [comment]
Ok, i made it

**Comment by taion** [comment]
Thanks!

**Pull request merged**